castai/resource_rebalancing_schedule.go

```diff
@@ -0,0 +1,340 @@
+package castai
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/castai/terraform-provider-castai/castai/sdk"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/samber/lo"
+)
+
+func resourceRebalancingSchedule() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceRebalancingScheduleCreate,
+		ReadContext:   resourceRebalancingScheduleRead,
+		DeleteContext: resourceRebalancingScheduleDelete,
+		UpdateContext: resourceRebalancingScheduleUpdate,
+		Importer: &schema.ResourceImporter{
+			StateContext: rebalancingScheduleStateImporter,
+		},
+		Description: "CAST AI rebalancing schedule resource to manage rebalancing schedules",
```

Comment:
Descriptions of this resource need a period at the end of the sentence.

Reply:
yeah it's weird as even golangci-lint is not complaining locally with kubecast settings:)